spec/classes/init_spec.rb

```diff
@@ -77,6 +77,21 @@
         it { is_expected.not_to contain_class('pki')}
         it { is_expected.to create_file('/etc/pki/simp_apps/grafana/x509')}
       end
+
+
+      context 'when ldap is false' do
+        let(:params) { {:ldap => false} }
+        it_behaves_like 'a structured module'
+        it { is_expected.to contain_class('grafana').with_ldap_cfg( {} ) }
+      end
+
+
+#      context 'when ldap is true' do
```

Comment:
any irregularities in tests should be thrown in pending blocks so we know something is not being tested with justification